www/_template/concepts/build-pipeline.md

```diff
@@ -3,30 +3,44 @@ layout: layouts/content.njk
+- When and how to use a bundler
+- Other options
 
-### Bundle for Production
+Snowpack’s philosophy is **You should be able to use a bundler because you want to, and not because you need to.** Snowpack treats bundling as an optional production optimization, which means you're free to skip the extra complexity of bundling until you need it.
 
-**You should be able to use a bundler because you want to, and not because you need to.** That was the original concept that Snowpack was designed to address. Snowpack treats bundling as an optional production optimization, which means you're free to skip over the extra complexity of bundling until you need it.
+Without a bundler, `snowpack build` generates a static production build of your site using the same unbundled approach as the `dev` command. For many projects, this is all you need.
 
-By default, `snowpack build` will build your site using the same unbundled approach as the `dev` command. This is fine for most projects, but you also may still want to bundle for production. Legacy browser support, code minification, code-splitting, tree-shaking, dead code elimination, and other performance optimizations can all be handled in Snowpack via bundling.
-
-Bundlers normally require dozens or even hundreds of lines of configuration, but with Snowpack it's just a one-line plugin with no config required. This is possible because Snowpack builds your application _before_ sending it to the bundler, so the bundler never sees your custom source code (JSX, TS, Svelte, Vue, etc.) and instead needs to worry only about building common HTML, CSS, and JS.
+Bundlers normally require dozens or even hundreds of lines of configuration, but adding a bundler to Snowpack is a one-line addition of a **bundler plugin** to `snowpack.config.js`. Snowpack then runs the bundler plugin automatically on `Snowpack build`.
 
 ```js
-// Bundlers plugins are pre-configured to work with Snowpack apps.
-// No config required!
+// snowpack.config.js
+// Example: enabling @snowpack/plugin-webpack
+// npm install —-save-dev @snowpack/plugin-webpack
 {
   "plugins": [["@snowpack/plugin-webpack"]]
 }
 ```
 
+TODO: how hard is it to rename bundler plugin to optimization plugin?
+
+Snowpack eliminates the need for heavy bundler configuration because it builds your application _before_ sending it to the bundler. The bundler then runs on the HTML, JS, and CSS prepared by Snowpack and never sees your custom source code (JSX, TS, Svelte, Vue, etc.).
+
+![build output example](/img/snowpack-build-example.png)
+
+Common uses for bundling are:
+- Legacy browser support
+- Code minification
+- Code-splitting
+- Tree-shaking
+- Dead code elimination (Todo: isn’t this covered in tree shaking?)
+
 See [our bundling guides](/guides/optimize-and-bundle) for more information about connecting bundled (or unbundled) optimization plugins for your production builds.
 
-## Legacy Browser Support
+## Legacy Browser Support - TODO move to guide
```

Comment:
This is still worth a mention here, or maybe as a sentence or two and then a link to the guide. I've heard plenty of people incorrectly say "I love snowpack but I need to support legacy browsers, so I can't use it"